gammapy/estimators/points/tests/test_lightcurve.py

```diff
@@ -634,3 +634,30 @@ def test_lightcurve_estimator_map_datasets():
     assert_allclose(table["norm_err"][0], [0.031508], rtol=1e-2)
     assert_allclose(table["counts"][0], [[2205, 2220]])
     assert_allclose(table["ts"][0], [2557.346464], rtol=1e-2)
+
+
+@requires_data()
+@requires_dependency("iminuit")
+def test_recompute_ul():
+    datasets = get_spectrum_datasets()
+    selection = ["all"]
+    estimator = LightCurveEstimator(
+        energy_edges=[1, 3, 30] * u.TeV, selection_optional=selection, n_sigma_ul=2
+    )
+    lightcurve = estimator.run(datasets)
+    table = lightcurve.to_table(format="lightcurve", sed_type="dnde")
```

Comment:
You might not need to convert to a table, but could directly access `light curve.dnde_ul.data`